source/configuration/read.txt

```diff
@@ -80,58 +80,46 @@ You can configure the following properties to read from MongoDB:
-            **Default**. Wraps the MongoSamplePartitioner and provides
-            help for users of older versions of MongoDB.
-
-       - ``MongoSamplePartitioner``
+       - ``SamplePartitioner``
              **Requires MongoDB 3.2**. A general purpose partitioner for
              all deployments. Uses the average document size and random
              sampling of the collection to determine suitable
              partitions for the collection. For configuration 
-             settings for the MongoSamplePartitioner, see
-             :ref:`conf-mongosamplepartitioner`.
+             settings for the SamplePartitioner, see
+             :ref:`conf-samplepartitioner`.
 
-       - ``MongoShardedPartitioner``
+       - ``ShardedPartitioner``
              A partitioner for sharded clusters. Partitions the
              collection based on the data chunks. Requires read access
              to the ``config`` database. For configuration settings for
-             the MongoShardedPartitioner, see
-             :ref:`conf-mongoshardedpartitioner`.
-
-       - ``MongoSplitVectorPartitioner``
-             A partitioner for standalone or replica sets. Uses the
-             :dbcommand:`splitVector` command on the standalone or the
-             primary to determine the partitions of the database.
-             Requires privileges to run :dbcommand:`splitVector`
-             command. For configuration settings for the
-             MongoSplitVectorPartitioner, see
-             :ref:`conf-mongosplitvectorpartitioner`.
-
-       - ``MongoPaginateByCountPartitioner``
-             A slow, general purpose partitioner for all deployments.
-             Creates a specific number of partitions. Requires a query
-             for every partition. For configuration settings for the
-             MongoPaginateByCountPartitioner, see
-             :ref:`conf-mongopaginatebycountpartitioner`.
+             the ShardedPartitioner, see :ref:`conf-shardedpartitioner`.
 
-       - ``MongoPaginateBySizePartitioner``
+       - ``PaginateBySizePartitioner``
              A slow, general purpose partitioner for all deployments.
              Creates partitions based on data size. Requires a query
              for every partition. For configuration settings for the
-             MongoPaginateBySizePartitioner, see
-             :ref:`conf-mongopaginatebysizepartitioner`.
+             PaginateBySizePartitioner, see 
+             :ref:`conf-paginatebysizepartitioner`.
+
+       - ``PaginateIntoPartitionsPartitioner``
+             A partitioner that creates a specified number of 
+             partitions for a collection. Uses the count and the 
+             average document size of a collection to calculate the 
+             number of documents to include in each partition. For 
+             configuration settings for the 
+             PaginateIntoPartitionsPartitioner, see
+             :ref:`conf-paginateintopartitionspartitioner`.
```

Comment:
Suggestion: although other entries use the form you used here, in the style guide, the preferred way to introduce additional reading is "To learn more about..."
This can help avoid repeating prepositions ("For X for Y, see...")

```suggestion
             number of documents to include in each partition. To learn more about the ``PaginateIntoPartitionsPartitioner``, see the :ref:`configuration settings description <conf-paginateintopartitionspartitioner>`.
```

Reply:
Sounds good, applied to all partitioner descriptions